.github/workflows/download-crowdin.yml

```
@@ -0,0 +1,19 @@
+name: Crowdin  download Action
+
+on:
+  schedule:
+    - cron: "20 3 * * *"
+  workflow_dispatch:
+
+jobs:
+  download-crowdin-exo:
```

Two remarks:
1. can you please delete usage of 'exo' prefix when possible in names, same for `CROWDIN_MAINTENANCE_EXO_VERSION`
2. what is the difference and impact with the files already [configured with gamification](https://github.com/Meeds-io/gamification/blob/develop/.github/workflows/upload-crowdin-main.yml) ?

```suggestion
  download-crowdin:
```

---

1. The actual concerned maintenance branches are eXo's. In the future, it may be needed if you will include meed's in maintenance
2. The gamification's may be changed to this one. As, the invoked scripts are centralized

---

ok thanks for the clarifications